rust/otap-dataflow/crates/otap/src/signal_type_router.rs

```diff
@@ -35,6 +39,94 @@ pub const PORT_METRICS: &str = "metrics";
 /// Name of the out port used for log signals
 pub const PORT_LOGS: &str = "logs";
 
+/// Metrics for the SignalTypeRouter processor.
```

Comment:
i left the metrics inside the STR module as it seems like a smaller set of values that may not grow too much.